charts/nextcloud/templates/service.yaml

```diff
@@ -13,6 +13,13 @@ spec:
   {{- if eq .Values.service.type "LoadBalancer" }}
   loadBalancerIP: {{ default "" .Values.service.loadBalancerIP }}
   {{- end }}
+  {{- if and .Values.service.loadBalancerClass (eq .Values.service.type "LoadBalancer") }}
+  loadBalancerClass: {{ .Values.service.loadBalancerClass }}
+  {{- end }}
+  {{- with .Values.service.loadBalancerSourceRanges }}
```

Comment:
Shouldn't there be a check if the value is even set? And also check if the service is of type LoadBalancer.

Reply:
Exactly